CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 Changelog
 =========
 
-# 0.44.3 / 2021-08-24 
+# 0.45.0 / Unreleased
```

Comment:
Should `Unreleased` be the current date? or is it updated later?

Reply:
I typically update on master once it's actually released.